Restructure Source Maps Upload page (#5799)
getsentry/sentry-docs: src/platform-includes/sourcemaps/upload/rollup/javascript.mdx

Sentry uses [releases](/product/releases/) to match the correct source maps to your events.
You can use the Sentry Rollup plugin to automatically create releases and upload source maps to Sentry when bundling your app.

## Installation

```npm
npm install @sentry/rollup-plugin --save-dev
```

```yarn
yarn add @sentry/rollup-plugin --dev
```

## Configuration

> Learn more about the configuration of the plugin using our [Sentry Rollup Plugin documentation](https://github.com/getsentry/sentry-javascript-bundler-plugins/tree/main/packages/rollup-plugin).

Reviewer: ```suggestion
Learn more about configuring the plugin in our [Sentry Rollup Plugin documentation](https://github.com/getsentry/sentry-javascript-bundler-plugins/tree/main/packages/rollup-plugin).
```

Example:

```javascript {filename:rollup.config.js}
import sentryRollupPlugin from "@sentry/rollup-plugin";

export default {
  input: "./src/index.js",
  plugins: [

>     // Put the Sentry esbuild plugin after any other plugins

Reviewer 2: ```suggestion
    // Put the Sentry rollup plugin after any other plugins
```

Reviewer: ```suggestion
    // Put the Sentry rollup plugin after all other plugins, to ensure no changes will be made to files after they're uploaded
```

    sentryRollupPlugin({
      org: "___ORG_SLUG___",
      project: "___PROJECT_SLUG___",

      // Specify the directory containing build artifacts
      include: "./out",

      // Auth tokens can be obtained from https://sentry.io/settings/account/api/auth-tokens/

>       // and needs the `project:releases` and `org:read` scopes

Reviewer: ```suggestion
      // and need `project:releases` and `org:read` scopes
```
